src/SymbolManager.cpp

```diff
@@ -339,6 +387,38 @@ void SymbolManager::labelChanged(QTreeWidgetItem* item, int column)
 	}
 }
 
+void SymbolManager::wipeSelectedItem()
+{
+	printf("wipeSelectedItem\n");
```

Comment:
Because if you deselect a label (ctrl+click the label for instance) or if you change the `Destination slot` and select the `Adress labels` tab, the old deprecated mappings of the controls will still appear in the `Location` and `Usage` tabs below. The user must select a new label to rewrite the mappings.